08-walkthrough-3.Rmd

```diff
@@ -194,15 +234,20 @@ rf_fit1 <- train(final_grade ~ .,
 rf_fit1
 ```
 
-The same tuning parameter values seem to be found with this method. Let's check just one last thing - what if we do not fix **min.node.size** to five?
+The same tuning parameter value seems to be found with this method: mtry 43. Let's check just one last thing - what if we do not fix **min.node.size** to five?
 
```

Comment:
one thing that I would find helpful here is more information on your thought process on resampling - what is the value of resampling? how do you choose methods//node sizes//etc? how many methods should you try?

Reply:
I still feel pretty novice at this, but I'll give an explanation a shot!!